atlp-rwanda/dynamites-ecomm-fe
# Fix image and routing

**Description:**

This issue addresses several improvements and bug fixes in the Admin Dashboard. The following tasks need to be completed:

**Tasks Completed**
**Admin Dashboard Image:**
- [ ] Ensure the image is correctly loaded and displayed.**
- [ ] Update the path or URL if the image source is incorrect.
- [ ] Verify that the image is responsive and displays correctly on different screen sizes.

**Redirect Issues:**

- [ ] Investigate and fix the redirect logic after certain actions (e.g., login or form submission).
- [ ] Ensure users are redirected to the correct pages without encountering errors.

**Toast Messages:**

- [ ] Implement toast messages for various user actions (e.g., form submission, login success/failure).
- [ ] Ensure the toast messages are user-friendly and informative.
- [ ] Verify that the toast messages are displayed consistently across the application.

**How Can be Manual Tested**
- open Hosted Link on render to check if image icons has appeared
- After signing up navigate  back 

**Related Issues**
#91
